subprojects/core/src/main/java/org/gradle/api/internal/plugins/DefaultPluginManager.java

```diff
@@ -320,7 +321,11 @@ public String getTargetPath() {
 
         @Override
         public String getBuildPath() {
-            return targetIdentifier.getBuildPath();
+            try {
+                return targetIdentifier.getBuildPath();
+            } catch (DefaultGradle.UnknownBuildPathException e) {
+                return "UNKNOWN";
```

Comment:
That should not be necessary.

Reply:
Yes. issue created in https://github.com/gradle/dotcom/issues/1435

Reply:
Can we make sure the issue in `dotcom` is addressed?